types/lodash/lodash-tests.ts

```diff
@@ -6068,20 +6156,45 @@ namespace TestMap {
     let array: number[] | null | undefined = [] as any;
     let list: _.List<number> | null | undefined = [] as any;
     let obj: any = {};
-    let dictionary: _.Dictionary<number> | null | undefined = obj;
+    let dictionary: _.Dictionary<number> | null | undefined = anything;
+    let numericDictionary: _.NumericDictionary<number> | null | undefined = anything;
 
     let listIterator: (value: number, index: number, collection: _.List<number>) => AbcObject = (value: number, index: number, collection: _.List<number>) => ({ a: 1, b: "", c: true });
     let dictionaryIterator: (value: number, key: string, collection: _.Dictionary<number>) => AbcObject = (value: number, key: string, collection: _.Dictionary<number>) => ({ a: 1, b: "", c: true });
+    let numericDictionaryIterator: (value: number, key: string, collection: _.NumericDictionary<number>) => AbcObject = (value: number, key: string, collection: _.NumericDictionary<number>) => ({ a: 1, b: "", c: true });
 
     {
         _.map(array);  // $ExpectType number[]
         _.map(array, listIterator);  // $ExpectType AbcObject[]
```

Comment:
I feel like this line isn't necessary anymore - it's a worse version of the test you do on the next line.